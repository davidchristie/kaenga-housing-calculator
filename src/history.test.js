import history from './history'

it('matches snapshot', () => {
  expect(history).toMatchSnapshot()
})
