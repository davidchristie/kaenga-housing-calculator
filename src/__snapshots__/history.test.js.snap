exports[`test matches snapshot 1`] = `
Object {
  "createHref": [Function],
  "createKey": [Function],
  "createLocation": [Function],
  "createPath": [Function],
  "getCurrentLocation": [Function],
  "go": [Function],
  "goBack": [Function],
  "goForward": [Function],
  "listen": [Function],
  "listenBefore": [Function],
  "push": [Function],
  "replace": [Function],
  "transitionTo": [Function],
  "unsubscribe": [Function],
}
`;
